src/main/java/com/openlattice/data/DataIntegrationApi.java

```diff
@@ -36,54 +39,21 @@
     String BASE                  = SERVICE + CONTROLLER;
     // @formatter:on
 
-    String ASSOCIATION = "association";
-
-    String COUNT                 = "count";
-    String DETAILED_RESULTS      = "detailedResults";
-    String EDGES                 = "edges";
-    String ENTITY_KEY_ID         = "entityKeyId";
-    String ENTITY_KEY_IDS        = "entityKeyIds";
-    String ENTITY_KEY_ID_PATH    = "{" + ENTITY_KEY_ID + "}";
+    String COUNT          = "count";
+    String ENTITY_KEY_IDS = "entityKeyIds";
     /**
      * To discuss paths later; perhaps batch this with EdmApi paths
      */
 
-    String ENTITY_SET            = "set";
-    String ENTITY_SET_ID         = "setId";
-    String SET_ID_PATH = "{" + ENTITY_SET_ID + "}";
-    String UPDATE      = "update";
-
-    String S3               = "s3";
-
-    @POST( BASE + "/" + ENTITY_SET + "/" + SET_ID_PATH )
-    IntegrationResults integrateEntities(
-            @Path( ENTITY_SET_ID ) UUID entitySetId,
-            @Query( DETAILED_RESULTS ) boolean detailedResults,
-            @Body Map<String, Map<UUID, Set<Object>>> entities );
+    String ENTITY_SET_ID  = "setId";
+    String UPDATE         = "update";
 
-    /**
-     * Creates a new set of associations.
-     *
-     * @param associations Set of associations to create. An association is the usual (String entityId, SetMultimap &lt;
-     * UUID, Object &gt; details of entity) pairing enriched with source/destination EntityKeys
-     */
-    @POST( BASE + "/" + ASSOCIATION + "/" + SET_ID_PATH )
-    IntegrationResults integrateAssociations(
-            @Body Set<Association> associations,
-            @Query( DETAILED_RESULTS ) boolean detailedResults );
-
-    @POST( BASE )
-    IntegrationResults integrateEntityAndAssociationData(
```

Comment:
Is there going to be a shuttle PR too? `integrateEntityAndAssociationData` is used in the shuttle test there

Reply:
oh good call, PR incoming, thanks!